### PR TITLE
chore(examples): introducing a socket.io service example

### PR DESCRIPTION
**Pull request by fl0wo**
A simple example of a Service used as a WebSocket server, and a mini front-end to show its synchronization of a multiplayer game.

Ideally, I'd like to maintain it by introducing more services like Redis.
Also as soon as [#1306](https://github.com/winglang/wing/issues/1306) is resolved I'd try to do a test deployment on AWS seeing it running on ECS.

### CODE REVIEW

**Comment by skorfmann** [comment]
Cool stuff! 

By default the CI assumes that `wing test --debug main.w` will have a test executed to tell if the example is working properly. Can you think of a test we could add here?

To disable AWS tests we can just add an empty `skip.ci.aws` file

File: examples/multiplayer-videogame/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "dependencies": {
+    "dgram": "^1.0.1",
+    "express": "^4.18.2",
+    "fs": "^0.0.1-security",
+    "http": "^0.0.1-security",
```
Comment (skorfmann) [inline examples/multiplayer-videogame/package.json:6]: why these packages ?

https://www.npmjs.com/package/http
https://www.npmjs.com/package/fs

I think that can be dropped.

File: examples/multiplayer-videogame/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+This is a multiplayer paintball game made with Wingcloud and VanillaJs.
+
+![img.png](images/infra2.png)
+
+## Infrastructure With Code
+
+The infrastructure is composed by:
+- A [cloud.Website](https://www.winglang.io/docs/standard-library/cloud/website) to host the website
+- A [cloud.Service](https://www.winglang.io/docs/standard-library/cloud/secret) as a websocket broadcasting server
```
Comment (skorfmann) [inline examples/multiplayer-videogame/README.md:17]:
```suggestion
- A [cloud.Service](https://www.winglang.io/docs/standard-library/cloud/service) as a websocket broadcasting server
```

**Review by skorfmann — CHANGES_REQUESTED** [review]

**Comment by fl0wo** [comment]
Introduced 5 tests that check whether service execution and counter work well together.

I'm fairly new to open-source contributions, so I hope I didn't screw up the commits 😅

**Comment by skorfmann** [comment]
Looks good, thanks! Gonna merge and fix potential ugprading issues locally.

**Pull request merged**